fiesta/apps/notifications/services/membership.py

```
@@ -0,0 +1,95 @@
+        _enqueue_editor_digests(membership=membership, section=section, config=config)
+
+
+def _send_member_received_email(*, membership: SectionMembership, section: Section) -> None:
+    preferences_url = f"https://{section.space_slug}.{settings.ROOT_DOMAIN}/notifications/preferences/"
+    context = {
+        "membership": membership,
+        "section": section,
+        "preferences_url": preferences_url,
+    }
+    send_notification_email(
+        subject=f"{section} – Application received",
```

Ruff (0.15.4)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)